src/sentry/static/sentry/app/views/organizationIntegrations/index.tsx

```diff
@@ -284,50 +291,52 @@ class OrganizationIntegrations extends AsyncComponent<
     );
 
     return (
-      <React.Fragment>
-        {!this.props.hideHeader && <SettingsPageHeader title={t('Integrations')} />}
-        <PermissionAlert access={['org:integrations']} />
-
-        <MigrationWarnings
-          orgId={this.props.params.orgId}
-          providers={this.providers}
-          onInstall={this.onInstall}
-        />
+      <DocumentTitle title={this.getTitle()}>
```

Comment:
Another approach could be to add a `DocumentTitle` component to our application that extends the library version, but accepts an `orgId` and and `withSentry` property in addition to the `title` property. That would remove the need for a module function and additional `getTitle` functions in each component. The resulting component could be used like

```jsx
<DocumentTitle title={t('Integrations')} orgId={organization.slug} withSentry />
```
I don't think it is necessary to wrap the entire view contents with `<DocumentTitle>` either. From looking at how [document title](https://github.com/gaearon/react-document-title/blob/56c71c934ad78aeed759b1972e6d332724176783/index.js#L29-L35) is built the child nodes are optional.

Reply:
@markstory ok, I'll check this, the main problem with `getTitle` that it's used on `AsyncView`s, so only other components which do not extend `AsyncView` would require some generic DocumentTitle generator. I ll try to do something like this today :)

Reply:
I think that sounds good @bsr3000 - I would make `withSentry` default true (or maybe change the param name to be `withoutSentry` and maybe name the component `SentryDocumentTitle` so people do not confuse it with `DocumentTitle`.

We can leave the `AsyncView` titles as is for now.

Reply:
@billyvg you are reading my thoughts  :)

```typescript
interface DocumentTitleProps {
    title: string;
    orgSlug: string;
    hasSentry: boolean;
}

const SentryDocumentTitle: FunctionComponent<DocumentTitleProps> = ({ title, orgSlug, hasSentry, children }) => {
    let _title = `${title} - ${orgSlug}`;
    if (hasSentry)
        _title = `${_title} - Sentry`;
        return (
            <DocumentTitle title={_title}>
                {children}
            </DocumentTitle>
        )
    };
```

it's only a scratch for now